addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -210,8 +210,8 @@ function factory(dependencies) {
          * hint `message-received`.
          */
         hasAutoScrollOnMessageReceived: attr(),
-        lastMessage: many2one('mail.message', {
```

Comment:
So I need to just bring back the field definition but I keep the one and all the dependencies changed ?

Reply:
Keep the existing field in case anyone used it by now (custo, other PR in progress, whatever) and make a new field that we use where needed. So this this is equivalent of keeping its definition but replace all its usage if that is what you are asking.